test/test_van/test_email.py

```diff
@@ -102,10 +152,33 @@ def test_get_email_messages(self, m):
     def test_get_email_message(self, m):
         m.get(
             self.van.connection.uri + "email/message/1",
-            json=mock_response[0],
+            json=mock_response_enriched[0],
             status_code=200,
         )
 
         response = self.van.get_email(1)
 
-        assert_matching_tables(response, mock_response[0])
+        assert_matching_tables(response, mock_response_enriched[0])
+
+    @requests_mock.Mocker()
+    def test_get_email_message_stats_agg(self, m):
+        m.get(
+            self.van.connection.uri + "email/messages",
+            json=mock_response_enriched,
+            status_code=200,
+        )
+        for resp in mock_response_enriched:
+            m.get(
+                self.van.connection.uri + f"email/message/{resp['foreignMessageId']}",
+                json=resp,
+                status_code=200,
+            )
+
+        response_t = self.van.get_email_stats(aggregate_ab=True)
+        response_f = self.van.get_email_stats(aggregate_ab=False)
+
+        assert len(response_t.to_dicts()) == 5
```

Comment:
Did not feel like hand-writing out the whole expected table here, so instead I just tested to see if we should get one more entry if we don't aggregate and ensure that we either aggregate or dis-aggregate on `recipientCount`.